endoreg_db/urls.py

```
@@ -6,14 +7,17 @@
     get_location_choices,
     get_morphology_choices, 
 )
-from .views.csrf import csrf_token_view
-#from .views.feature_selection_view import FetchSingleFramePredictionView // its implemented in ando-ai other project need to add here
+from .views.Frames_NICE_and_PARIS_classifications_views import ForNiceClassificationView, ForParisClassificationView
+# endoreg_db_production/endoreg_db/urls.py
+from .views.keycloak_views import VideoView, keycloak_login, keycloak_callback, public_home
+#from .views.feature_selection_view import FetchSingleFramePredictionView // its implemented in endo-ai other project need to add here
 from .views.video_segmentation_views import VideoView, VideoLabelView,UpdateLabelSegmentsView
```

Ruff (0.8.2)

Redefinition of unused `VideoView` from line 12

(F811)